persistence-api/src/main/java/io/stargate/db/schema/Column.java

```diff
@@ -955,6 +960,17 @@ public Order order() {
     return null;
   }
 
+  @Override
```

Comment:
What about caching? Are we setting the `@Lazy` or `@Derived` here? We could actually only do it on the keyspace level as that's what we will be calling over and over again?

Reply:
That's a good question. I only added it to the keyspace level because that's where it's repeatedly checked. I'm fine with adding  `@Values.Derived` to the rest of the sub-objects.

Reply:
I added it to the more expensive schema types.